src/clients/lambda_function.py

```diff
@@ -0,0 +1,95 @@
+# Python's libraries
+from typing import Dict
+from typing import Any
+
+# Own's modules
+from client_modules.dao.client_dao import ClientDAO
+from client_modules.models.client import HIBerryClient
+from client_modules.utils.doorman import DoormanUtil
+from client_modules.errors.auth_error import AuthError
+from client_modules.data_mapper.client_mapper import ClientHelper
+
+
+# Third-party libraries
+from pydantic.error_wrappers import ValidationError
+from aws_lambda_powertools import Logger
+from aws_lambda_powertools.utilities.typing import LambdaContext
+
+
+def create_client(event: Dict[str, Any], context: LambdaContext) -> Dict[str, Any]:
+    """This function is the entry point of this process that wil receive a payload as an input
+    an will attempt to create an entry in DynamoDB.
+
+    :param event: Custom object that can come from an APIGateway
+    :type event: Dict
+    :param context: Regular lambda function context
+    :type context: LambdaContext
+    :return: Custom object with the reponse from the lambda, it could be a 201, if the resource was created
+    or >= 400 if theras was an error
+    :rtype: Dict
+    """
+
+    logger = Logger()
+    logger.info("Initializing Create Client function")
+    doorman = DoormanUtil(event, logger)
+
+    try:
+        username = doorman.get_username_from_context()
+        is_auth = doorman.auth_user()
+        if is_auth is False:
+            raise AuthError(f"User {username} is not authorized to create a client")
+
+        body = doorman.get_body_from_request()
+
+        logger.debug(f"Incoming data is {body=} and {username=}")
```

Comment:
why the =?

Reply:
This is a self documenting expression from f-string, it will evaluate string as 
Incoming data is body=<body> and username=<username>

https://docs.python.org/3/whatsnew/3.8.html#f-strings-support-for-self-documenting-expressions-and-debugging